afs/templates/javascript.htm

```diff
@@ -15,8 +15,8 @@
     upload_dataset_select_dataset_files_step = "{% url 'upload_dataset_select_dataset_files_step' %}"
     upload_dataset_file_renderer = '(tileid)=>{return "{% url "upload_dataset_file_renderer" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", tileid)}',
     format_render_map = '(format)=>{return "{% url "format_render_map" "aaaaaaaa" %}".replace("aaaaaaaa", format)}'
+    add_chemical_analysis_images_file_upload = "{% url 'add_chemical_analysis_images_file_upload' %}"
     download_project_files = "{% url 'download_project_files' %}"
-    temp_file = "{% url 'temp_file' %}"
```

Comment:
Why is this URL getting removed? It seems related to the report workflow rather than this one.